src/browsers/tob/shared/components/ViolinPlot.js

```diff
@@ -118,7 +117,20 @@ const ViolinPlot = ({ width, data, height, margin, categoryColors }) => {
             </g>
           ))}
           {boxplotStats.map((item, i) => (
-            <g key={`BoxStat${cellLines[i]}`} transform={`translate(${xScale(cellLines[i])}, 0)`}>
+            <g
+              key={`BoxStat${cellLines[i]}`}
+              transform={`translate(${xScale(cellLines[i])}, 0)`}
+              onMouseEnter={(e) => {
+                setHoveredData({
+                  value: item,
+                  x: e.screenX - 125,
```

Comment:
Not sure screenX and screenY are the ones we want, especially if the browser is not maximised. I think it breaks but will have to check this

Reply:
I switch to `clientX` and `clientY`